contracts/cosmwasm-vm/README.md

```diff
@@ -137,6 +148,9 @@ IBC-Core is the reference implementation of the Inter-Blockchain Communication (
 
 The icon-light-client in IBC enables blockchain networks to communicate with each other without having to trust each other. This is achieved by using the cryptographic proofs to verify that transactions are valid and have been executed correctly.
 
+#### [cw-xcall-ibc-connection](./cw-xcall-ibc-connection/src/) :
+
```

Comment:
May be we should add here 1-2 lines for this contract description
cc: @ibrizsabin

Reply:
ya i think i missed it. I have provided a short desc to deepak  to update.